test/guest/fedora.build

```diff
@@ -28,9 +28,10 @@ fi
 out=$1
 os=$2
 arch=$3
+test -z "$VOLUME_SIZE" && VOLUME_SIZE="8G"
```

Comment:
If we use this, should we rename the variable to something like `BOOTSTRAP_VOLUME_SIZE`?

I'm not sure if `VOLUME_SIZE` conflicts with anything else, it seems pretty generic. Also, we only use this in the fedora-like bootstrap now.